src/extension/app.css

```diff
@@ -1037,3 +1086,35 @@
 .hlx-sk-overlay .modal.modal-logout-error::before {
   content: "Sign out failed. Please try again later.";
 }
+
+.hlx-sk .feature-container .info .page-info .edit-date::before {
+  content: "Edited: ";
+}
+
+.hlx-sk .feature-container .info .page-info .preview-date::before {
+  content: "Previewed: ";
+}
+
+.hlx-sk .feature-container .info .page-info .publish-date::before {
+  content: "Published: ";
+}
+
+.hlx-sk .feature-container .info .page-info .now::before {
+  content: "Now";
+}
+
+.hlx-sk .feature-container .info .page-info .seconds-ago::before {
+  content: " Seconds ago";
+}
+
+.hlx-sk .feature-container .info .page-info .minutes-ago::before {
+  content: " Minutes ago";
+}
+
+.hlx-sk .feature-container .info .page-info .today::before {
+  content: "Today at ";
+}
+
+.hlx-sk .feature-container .info .page-info .yesterday::before {
+  content: "Yesterday at ";
+}
```

Comment:
I'd probably add a placeholder for the value since the position of the value may not be at the same position in every language.

Reply:
Discussed with @rofe.  For now that is not possible via css, so will leave as is.